### ISSUE TITLE
03_pytorch_computer_vision.ipynb — baseline model

### ISSUE THREAD

**Issue by sascharo** [mrdbourke/pytorch-deep-learning#743]
Why don't we use a softmax function to turn the logits into prediction probabilities here?

**Comment by nickyreinert** [comment]
It's a little confusing, because at some points there's an additional softmax, at some not.  don't know if Daniel explained it at some point: Argmax can process logits to return labels, it does not need probabilites here.

You can check it for yourself:
```

print(f"logits: {y_logits[1]}") 
print(f"Probabilits: {y_preds_probabilities[1]}") # softmax'ed
print(f"Labels: {y_preds[1]}") # argmax'ed
print(f"Argmax'ed: {y_logits.argmax(dim=1)[1]}") # argmax'ed
print(f"Softmax'ed: {y_logits.softmax(dim=1)[1]}") # argmax'ed
print(f"Softmax'ed then Argmax'ed: {y_logits.softmax(dim=1).argmax(dim=1)[1]}") # argmax'ed
```

So, if you need probabilites, for what reason ever, you may you **softmax**. But for the actual result, like the labels, **argmax** is sufficient.

(There are probably other reasons or scenarios, I may be corrected, extended)
